### ISSUE TITLE
Unable to login to Azure

### ISSUE THREAD

**Issue by RSchama** [microsoft/vscode-tools-for-ai#1562]
Does this occur consistently? 
Repro steps:

1.
2.

Action: azureAccount.onSubscriptionsChanged
Error type: 123
Error Message: Unknown Error retrieving susbcriptions from Azure Account extension

Version: 0.8.2
OS: darwin
OS Release: 21.4.0
Product: Visual Studio Code
Product Version: 1.66.2
Language: en

<details>
<summary>Call Stack</summary>

```
b.<anonymous> extension.js:2:1989015
s extension.js:2:1985916
```

</details>

**Comment by tbombach** [comment]
Hi @RSchama  thank you for reporting this! We'll look into it and I apologize for the trouble. Would you mind sharing the actions you were taking when you saw this error message? Did it prevent you from using the AML extension or other Azure extensions?

**Comment by jedcheng** [comment]
Hi there. Are you still having this issue?

This problem can be got around by signing in through the Azure plugin, instead of the Azure ML plugin.

Nonetheless, I got in touch with a Microsoft engineer about the exact same problem last month (on a Mac and signing in with Chrome).

After that, it seems that there is a recent update with VS code or Electron (I am not sure) that fixed this problem for signing in with Chrome.

We then tested it for Safari and Chrome to sign in. Using Safari doesn't help but using Chrome does work.
